src/widgets/answers/answers.tsx

```diff
@@ -53,20 +53,20 @@ export type AnswersTemplates = {
   /**
    * Template to use for the header. This template will receive an object containing `hits` and `isLoading`.
    */
-  header: Template<{
+  header?: Template<{
```

Comment:
We can `AnswersTemplates = Partial<{/* ... */}>` like elsewhere? (it's less error prone)

Reply:
yes, I agree, we should probably have all templates as non-optional in the template type and use Partial in the parameter (nothing in the component needed as change then)

Reply:
Since it's unanimous, I'll do a stacked PR that converts all types to `export type T = Partial<{ ... }>`  once this one is finished

Reply:
could you do the stacked PR before we merge this one? That way there's less churn on master

Reply:
I'll do a stacked PR on https://github.com/algolia/instantsearch.js/pull/4791 right after!

Reply:
thought about it again, and since it touches the css types too, let's do a completely separate PR, no need to worry about master churn in this case :)